diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+Comfort:2
+clarity:3
+I used google and the lessons to help
+
```

Comment:
Try to put your comfort/clarity in your pull request comment and/or final commit.